src/commands/gaming/Wine/Stable.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh -e
+
+. ../../common-script.sh
+
+installDependencies() {
+
+    echo -e "${YELLOW}Installing dependencies...${RC}"
```

Comment:
```suggestion
    echo -e "${YELLOW}Enabling Multi-Lib...${RC}"
```
This should be changed to "Enabling Multi-Lib..."